web-quiz-app/src/pages/EditQuiz.jsx

```diff
@@ -91,6 +98,19 @@ function EditQuiz(){
             })
         }
     }
+
+    function handleActiveSwitch() {
+        setActiveSwitch(!activeSwitch)
+
+        if(quizID) {
+            QuizDataService.changeQuizActivity(quizID)
```

Comment:
Just out of curiosity, if we aren't using the response from this method, would we still need the `.then` portion?